fastlane/Fastfile

```diff
@@ -0,0 +1,86 @@
+system("cd .. && npm install shelljs@0.7.0")
+system("cd .. && ./install.sh")
+
+lane :All do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SalesforceSDK:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest :libs:SalesforceReact:assembleDebug :native:NativeSampleApps:RestExplorer:connectedAndroidTest",
+flags: "--continue --no-daemon --profile --stacktrace"
+)
+end
+
+lane :PR do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest :libs:SalesforceReact:assembleDebug",
```

Comment:
My concern is that every PR build will fail, and the people will easily to think the failure is a known one always, unless to check the results every single time. we will do that? And after each change pushed into dev branch, there will be a full test pass run automatically triggered on all supported versions, including sdk test on 19. In this case, we still don't lose the control. It is not a idea solution, while balance the effort and the coverage, I would prefer not include sdk test in our PR, until we figure a way to solve that emulator issue.